README.md

```diff
@@ -1,6 +1,6 @@
 # Address Book (Level 4)
 
-[![Build Status](https://travis-ci.org/se-edu/addressbook-level4.svg?branch=master)](https://travis-ci.org/se-edu/addressbook-level4)
+[![Build Status](https://travis-ci.org/CS2103JAN2017-T09-B3/main.svg?branch=master)](https://travis-ci.org/se-edu/addressbook-level4)
```

Comment:
errr.. you only changed the link to the badge. the actual link is still point to se-edu :-P